frontend/packages/integration-tests-cypress/tests/cluster-settings/update-modal.cy.ts

```diff
@@ -53,21 +54,24 @@ describe('Cluster Settings cluster update modal', () => {
     ).as(CLUSTER_VERSION_ALIAS);
     cy.wait(`@${CLUSTER_VERSION_ALIAS}`, WAIT_OPTIONS);
     clusterSettings.openUpdateModalAndOpenDropdown();
-    cy.byTestID('update-cluster-modal')
-      .find('[data-test="dropdown-with-switch-switch"]')
-      .should('exist')
-      .click({ force: true });
     cy.byTestID('update-cluster-modal')
       .find('[data-test="dropdown-with-switch-menu-item-4.17.1"]')
       .should('exist')
       .click();
     cy.byTestID('update-cluster-modal-not-recommended-alert').should('not.exist');
-    cy.byTestID('update-cluster-modal').find('[data-test="dropdown-with-switch-toggle"]').click();
-    cy.byTestID('update-cluster-modal')
-      .find('[data-test="dropdown-with-switch-menu-item-4.16.4"]')
-      .should('exist')
-      .click();
-    cy.byTestID('update-cluster-modal-not-recommended-alert').should('exist');
+    // Only run locally as this is flaking in CI
```

Comment:
Is there a bug tracking this flaky test so we don't lose track of it?

Reply:
Ignore that

Reply:
Was planning on keeping this bug as the tracker.  Just trying to improve things in CI ATM.